modules/ROOT/pages/cryptography-troubleshooting.adoc

```diff
@@ -0,0 +1,96 @@
+= Troubleshoot Cryptography Module
+
+To troubleshoot Cryptography module, become familiar with the information about app logs, PGP protocol, attached and detached signature, and interpreting commonly thrown messages.
+
+== View the App Log
+
+If you encounter problems while running your Mule runtime engine (Mule) app, check the app log as follows:
+
+* If you’re running the app from Anypoint Platform, the output is visible in the Anypoint Studio console window.
+* If you’re running the app using Mule from the command line, the app log is visible in your OS console.
+
+Unless the log file path is customized in the app’s log file `log4j2.xml`, view the app log in the default location `MULE_HOME/logs/<app-name>.log`.
+
+== Enable Cryptography Module Debug Logging
+
+To begin troubleshooting Cryptography Module, enable debug logging to see the exact error messages:
+
+. Access Anypoint Studio and navigate to the *Package Explorer* view.
+. Open your application’s project name.
```

Comment:
You don't open the name; you open the app by clicking (?) the name. GLOBAL/boilerplate:
```suggestion
. Open your application by clicking its project name.
```
clicking, selecting, depending on the control.